bin/ruby-build

```diff
@@ -1256,6 +1256,29 @@ build_package_ldflags_dirs() {
   done
 }
 
+build_package_enable_yjit() {
+  local rustc_installed rustc_ver
+
+  rustc_installed=""
+  if type rustc &>/dev/null; then
```

Comment:
Could you use `type -p rustc`? That's what's used in other places.

Reply:
Sure. I had actually copied a different location that did *not* use -p (search for "type shasum"). But I'm not opposed to that. I couldn't find documentation on type easily since it's a bash builtin and the man page is horrible for searching. But it's easy enough to add.